.husky/pre-commit

```diff
@@ -1,6 +1,3 @@
-#!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
-
```

Comment:
Why have you removed this? Or what was this for in the first place?

Reply:
See https://github.com/typicode/husky/releases/tag/v9.0.1 > How to migrate